source/blog/2018-12-07-the-ember-times-issue-76.md

```diff
@@ -22,8 +22,18 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Jingle bells 🔔, jingle bells 🔔, ✨ DecEmber 🐹🎄 all the way!](https://www.emberjs.com/blog/2018/11/29/december-event.html)
+
+Find one of the issues 🔍
```

Comment:
Do you want these to be on different lines? Not sure how to make that show up at first glance

Reply:
Yes exactly, thank you for checking! Here's the update :sparkles:

<img width="881" alt="capture d ecran 2018-12-06 a 19 42 30" src="https://user-images.githubusercontent.com/8811742/49604902-6aaa8600-f98f-11e8-9382-56a39e9a8e53.png">